book/Makefile

```diff
@@ -9,20 +9,23 @@ MDBOOK_LINKCHECK_VERSION = 0.7.7
 MDBOOK_MERMAID_VERSION = 0.12.6
 MDBOOK_TOC_VERSION = 0.14.1
 
+
+all: deps check build serve
+
 #
-# use `make deps` to install the dependencies required to serve or build the book
+# Installs the dependencies required to serve or build the book
 #
 
 deps:
 	cargo install "mdbook@$(MDBOOK_VERSION)"
 	cargo install "mdbook-admonish@$(MDBOOK_ADMONISH_VERSION)"
 	cargo install "mdbook-katex@$(MDBOOK_KATEX_VERSION)"
-	# cargo install "mdbook-linkcheck@$(MDBOOK_LINKCHECK_VERSION)"
+	cargo install "mdbook-linkcheck@$(MDBOOK_LINKCHECK_VERSION)"
 	cargo install "mdbook-mermaid@$(MDBOOK_MERMAID_VERSION)"
 	cargo install "mdbook-toc@$(MDBOOK_TOC_VERSION)"
 
 #
-# use `make check` to check if your installed dependencies match what we've listed above
+# Checks if your installed dependencies match what we've listed above
```

Comment:
what command does this now instead? If it is not done automatically, I would keep the comment just in case someone wants to check this manually. Same with the other comments that have been changed in the Makefile.

Reply:
The same command below, `make check`. It's just that everything in makefile was duplicated, there was a target "`mytarget`" and a comment "use `make mytarget` to ...".

If it clarified but not confused things I can return it back?